pkg/ctl/generate/generate.go

```diff
@@ -0,0 +1,14 @@
+package generate
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/weaveworks/eksctl/pkg/ctl/cmdutils"
+)
+
+// Command creates `generate` commands
+func Command(flagGrouping *cmdutils.FlagGrouping) *cobra.Command {
+	verbCmd := cmdutils.NewVerbCmd("generate", "Generate GitOps manifests", "")
+	cmdutils.AddResourceCmd(flagGrouping, verbCmd, generateProfileCmd)
```

Comment:
Unrelated to the PR but I think `AddResourceCmd` should simply be called `AddCmd` since it's also used for non-resources.